include/solvers/flow_control.h

```diff
@@ -57,104 +153,29 @@ class FlowControl
   double         beta_n1     = 0;
   double         flow_rate_n = 0;
   double         flow_rate_1n;
-  double         area = 0;
+  double         area;
```

Comment:
Do not give default value to variables like that outside of the constructor. This should be done in the constructor. If you want to set them to zero, set them in the constructor itself.